.github/workflows/system-tests.yml

```diff
@@ -188,12 +192,12 @@ jobs:
         run: ./run.sh PARAMETRIC
 
       - name: Compress artifact
-        if: needs.needs-run.outputs.outcome == 'success' || github.event_name == 'schedule'
+        if: always() || github.event_name == 'schedule'
         run: tar -czvf artifact.tar.gz $(ls | grep logs)
 
       - name: Upload artifact
         uses: actions/upload-artifact@v3
-        if: needs.needs-run.outputs.outcome == 'success' || github.event_name == 'schedule'
+        if: always() || github.event_name == 'schedule'
```

Comment:
This causes builds to fail on main, and I'd expect it to do the same on 2.7.

Reply:
@tabgok Can you address this and fix the merge conflicts?

Reply:
removed these changes and fixed any other merge conflicts 👍